CHANGELOG.md

```diff
@@ -5,6 +5,14 @@
 ### Features
 
 - [#413](https://github.com/okta/okta-auth-js/pull/413) Adds support for Typescript. Uses named exports instead of default export.
+- [#444](https://github.com/okta/okta-auth-js/pull/444) New method `tokenManager.hasExpired` to test if a token is expired
+
+### Breaking Changes
+
+- [#444](https://github.com/okta/okta-auth-js/pull/444)
+  - Implements "active" autoRenew. If autoRenew is true, tokens will be renewed before expiration (without calling `tokenManager.get`). If autoRenew is false, tokens will be removed from storage on expiration.
+  - `onSessionExpired` option has been removed
+  - `tokenManager.get` no longer implements autoRenew functionality. It is possible that the token returned from the TokenManager may be expired. New method `tokenManager.hasExpired` can be used to test the token.
 
 ## PENDING
```

Comment:
Just noting that the PENDING part will have to be fixed before we release.  I assume you're on that, but I wanted a comment so it doesn't get overlooked.